core/src/main/res/values/strings.xml

```diff
@@ -719,6 +719,9 @@
     <string name="keep_updated">Keep Updated</string>
     <string name="keep_updated_summary">Include this podcast when (auto-)refreshing all podcasts</string>
     <string name="auto_download_disabled_globally">Auto download is disabled in the main AntennaPod settings</string>
+    <string name="podcast_listendes_for">Listened for :</string>
+    <string name="podcast_on_device">Episodes on the device :</string>
```

Comment:
Please remove the space character between the text and the colon (`:`)

I think names like `statistics_listened_for` and `statistics_episodes_on_device` would be more clear.